app/controllers/admin/collections_controller.rb

```diff
@@ -298,6 +299,12 @@ def update_access(collection, params)
 
     collection.default_visibility = params[:visibility] unless params[:visibility].blank?
     collection.default_hidden = params[:hidden] == "1"
+    collection.default_lending_period = build_default_lending_period
+  end
+
+  def build_default_lending_period
+    iso_duration = "P#{params["add_default_lending_period_days"]}DT#{params["add_default_lending_period_hours"]}H"
+    int_duration = iso_duration.is_a?(Integer) ? ActiveSupport::Duration.parse(iso_duration).to_i : Settings.controlled_digital_lending.default_lending_period.to_i
```

Comment:
What about something like:
```suggestion
    collection.default_lending_period = build_default_lending_period.to_i
  end

  def build_default_lending_period
    params["add_default_lending_period_days"].days + params["add_default_lending_period_hours"].hours
  rescue
    Settings.controlled_digital_lending.default_lending_period
```